doc/source/whatsnew/v0.15.0.rst

```diff
@@ -412,7 +412,7 @@ Rolling/Expanding Moments improvements
 
     s = pd.Series([10.5, 8.8, 11.4, 9.7, 9.3])
 
-  Behavior prior to 0.15.0:
+  behavior prior to 0.15.0:
```

Comment:
not sure about this one, I guess you have a reason, but looks correct as it was.